### DISCUSSION TITLE
Trying to find some way to communicate about grossly incorrect information on https://dortania.github.io/OpenCore-Legacy-Patcher/

### DISCUSSION THREAD

**Discussion by Dech01** [community/community#102001] [New to GitHub]
Their info. on how to do updates to Mac OS is egregiously incorrect - it took me four hours of hard work to fix the issue; "Google Bard" had no idea and if it had access to the Dortania site it would have had what I had; worse than no idea; a false idea.  There are links on that site to "help us improve" - they aren't that at all and a telephone assist line went to a YT video of Rick Astley.

**Comment by queenofcorgis** [comment]
Hi @Dech01 thanks for posting in the community 🎉 

GitHub Community doesn't provide support or assistance for projects that are hosted on GitHub, but I do have a few pointers. 

That web address (https://dortania.github.io/OpenCore-Legacy-Patcher/) likely contains their username or org name in it (the bit before `github.io`) so in this case it would be `dortania` - if you were to type the web address `github.com/dortania` you'd be sent to their org or profile page depending on the type of account dortania is. From there you can see if there are repositories that match the second half of the URL you shared `OpenCore-Legacy-Patcher`.

Once you locate the repository, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
